src/React/Redux.purs

```diff
@@ -87,30 +86,27 @@ instance semigroupReducer :: Semigroup state' => Semigroup (Reducer action state
 instance monoidReducer :: Monoid state' => Monoid (Reducer action state state') where
   mempty = Reducer (const (const (mempty)))
 
-createProviderElement :: forall action props props' state. Store action state -> ReduxReactClass state props props' -> props -> Array React.ReactElement -> React.ReactElement
-createProviderElement store reduxClass props children = React.createElement providerClass { store } [ createElement reduxClass props children ]
-
-createProviderElement_ :: forall action props state. Store action state -> ReduxReactClass' state props -> React.ReactElement
-createProviderElement_ store reduxClass = React.createElement providerClass { store } [ createElement_ reduxClass [] ]
-
-connect :: forall props props' state. Getter' (Tuple state props) props' -> React.ReactClass props' -> ReduxReactClass state props props'
+connect :: forall props props' state. Getter' (Tuple state props) props' -> React.ReactClass props' -> ReactClass state props props'
 connect slens class_ = runFn3 connectFn Tuple (view slens) class_
 
-connect_ :: forall props state. Getter' state props -> React.ReactClass props -> ReduxReactClass' state props
+connect_ :: forall props state. Getter' state props -> React.ReactClass props -> ReactClass' state props
 connect_ slens class_ = connect slens' class_
   where
   slens' :: Getter' (Tuple state Unit) props
   slens' = to (view slens <<< fst)
```

Comment:
why not `slens <<< _1` `_1` is in `Data.Lens.Lens.Tuple`

Reply:
I think you are looking at an outdated version here. `connect` does not use `Getter'` in the latest version.